exporter/clickhousemetricsexporter/clickhouse.go

```diff
@@ -313,8 +314,14 @@ func (ch *clickHouse) Write(ctx context.Context, data *prompb.WriteRequest) erro
 			}
 		}
 
-		return statement.Send()
-
+		start := time.Now()
+		err = statement.Send()
```

Comment:
@srikanthccv is `statement.Send()` a blocking call?

Reply:
Yes